docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,176 @@
 # S3cmd with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-`S3cmd` is a CLI client for managing data in AWS S3, Google Cloud Storage or any cloud storage service provider that uses the s3 protocol.  `S3cmd` is open source and is distributed under the GPLv2 license.
+`S3cmd` is an open source CLI client for managing data in AWS S3, Google Cloud Storage, or any cloud storage service provider that uses the **s3** protocol. `S3cmd` is distributed under the [GPLv2 License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html).
 
-In this recipe we will learn how to configure and use S3cmd to manage data with Minio Server.
+This guide describes how to configure `S3cmd` and use it to manage data on Minio Server. These are the steps you will follow:
 
-## 1. Prerequisites
+1. [Install Minio Server](#installminioserver) 
+2. [Install `S3cmd`](#installs3cmd) 
+3. [Configure `S3cmd`](#configures3cmd) 
+4. [Examples of Typical `S3cmd` Commands](#runs3cmdcommands)
```

Comment:
unnecessary capitalization of Typical.

Reply:
@deekoder I would disagree in this case since it is a heading and this is standard practice for headings.